src/App.tsx

```diff
@@ -2,15 +2,19 @@ import "./App.css";
 import Example from "./example/counter";
 import { Provider } from "react-redux";
 import store from "./store";
-import Application from "./example/dynamic-routing/components/Application";
+import { ThemeProvider } from "styled-components";
+import theme from "./example/theme/Colors";
+import { GlobalStyle } from "./example/global-styles";
 
 function App() {
   return (
     <Provider store={store}>
-      <div className="App">
-        <Application />
-        <Example />
-      </div>
+      <ThemeProvider theme={theme}>
+        <GlobalStyle />
```

Comment:
Give this a meaningful name, like main layout or something, plus does <Example /> component should also be a part of global theme right?

Reply:
yes hamza bhai, its a part of our components.